faq/inactive-users-detection.md

```diff
@@ -0,0 +1,52 @@
+---
+description: This article explains how Inactive Users are detected and defined within Syskit Point.
+---
+
+# Inactive Users
+
+**Inactive Users** across your Microsoft 365 environment are **detected to help you optimize your license usage** and maintain a **safe and healthy environment**. 
+
+This article explains **how a user's activity is determined based on the user type** and how you can define any available **inactivity thresholds in Syskit Point's Settings**. 
+
+## Determining a User's Activity
+
+Syskit Point determines a user's last activity based on **sign-in data from Microsoft 365 and Azure AD, depending on the user type**.
+
+### Internal (Organization) Users
+
+For **Internal Users**, activity is determined using the **Unified Audit Log**, which includes events such as **the last date the user logged into their Microsoft 365 account or e-mail**.
+
+This **applies to regular and external users that still appear in your organization's Azure AD directory**. 
+
+### Guest Users
```

Comment:
Mislim da bi bilo zgodno navesti i licensing requirement da možemo do tih logova uopće: https://learn.microsoft.com/en-us/entra/identity/monitoring-health/howto-manage-inactive-user-accounts?tabs=admin-center

Vidim i u kodu ovaj msg kod logika za dohvat tih vrijednosti: "Tenant must have a premium license for fetching of signIns to work"